nodes/cma/nodes-cma-autoscaling-custom.adoc

```diff
@@ -49,3 +49,21 @@ Successfully updated ScaleTarget
 ----
 
 You can temporarily pause the autoscaling of a workload object, if needed. For example, you could pause autoscaling before performing cluster maintenance.
+
+.Custom metrics autoscaler workflow
+image::564_OpenShift_Custom_Metrics_Autoscaler_0224.png[Custom metrics autoscaler workflow]
+
+1. A user creates or modifies a scaled object custom resource for a workload on the cluster. The object contains the scaling configuration for that workload. Prior to accepting the new object, the OpenShift API server sends it to the custom metrics autoscaler admission webhooks process to ensure that the object is valid. If validation succeeds, the API server persists the object.
```

Comment:
```suggestion
1. A user creates or modifies a scaled object custom resource for a workload on a cluster. The object contains the scaling configuration for that workload. Prior to accepting the new object, the OpenShift API server sends it to the custom metrics autoscaler admission webhooks process to ensure that the object is valid. If validation succeeds, the API server persists the object.
```
I would say to use an indefinite article (`a cluster`) unless the sentence is referring to a specific cluster